starter-repos/anthos-config-management/cluster/app-operator-role.yaml

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
I am curious how you came up these role definitions?
Does the ACM team have recommendations on the required roles?

Reply:
These roles are based on the following requirements and assumptions.
- (requirement) Operator need to apply reposync in the namespace, apply secret, apply rolebinding.
- (assumption) The application being deployed contain some k8s resources such as Deployment, Service, ConfigMap, Secret

Reply:
I see. Thanks for the explanation.